test/e2e/azure_test.go

```diff
@@ -844,6 +844,135 @@ var _ = Describe("Workload cluster creation", func() {
 		})
 	})
 
+	Context("Creating an AKS cluster using ClusterClass [Managed Kubernetes]", func() {
+		It("with a single control plane node, one linux worker node, and one windows worker node", func() {
+			// use default as the clusterclass name so test infra can find the clusterclass template
+			os.Setenv("CLUSTER_CLASS_NAME", "default")
+
+			// use "cc" as spec name because natgw pip name exceeds limit.
```

Comment:
```suggestion
			// Use "cc" as spec name because NAT gateway pip name exceeds limit.
```

What does pip mean?

Reply:
I think it stands for public ip but I'm not 100% sure. Can you confirm @CecileRobertMichon?